### DISCUSSION TITLE
JsonNullable support?

### DISCUSSION THREAD

**Discussion by Ziaunys** [quarkiverse/quarkus-openapi-generator#807] [Ideas]
Previously I have used the openapi-generator jax-rs generator with its Jackson JsonNullable support. See:
 https://github.com/OpenAPITools/jackson-databind-nullable

It would be really nice if that was supported here. Otherwise I don't know how to represent the situation where a JSON field isn't present vs. present and null. If there's currently a way to achieve the same behavior I'm interested in knowing how to do this. Thank you.

**Comment by ricardozanini** [comment]
@Ziaunys can you paste here an OpenAPI spec example? I'm pretty sure we already support this use case. 🤔
